src/components/SubscribeButton/SubscribeButton.tsx

```diff
@@ -40,11 +32,18 @@ const _SubscribeButton = ({
         }
       >
         <Button
-          className={twMerge('p-0', className)}
+          className={twMerge(
+            'flex items-center justify-center gap-3 p-0',
+            className,
+          )}
           type="text"
           onClick={onSubscribeButtonClicked}
+          loading={loading}
+          icon={<SubscribeButtonIcon isSubscribed={isSubscribed} />}
         >
-          <SubscribeButtonIcon isSubscribed={isSubscribed} />
+          <span className="font-base text-sm">
+            <Trans>Get Updates</Trans>
```

Comment:
WDYT about `Watch` instead? Feel like this is common enough language these days.